pkg/query-service/app/integrations/thirdPartyApi/translator.go

```diff
@@ -61,7 +61,7 @@ func BuildDomainList(thirdPartyApis *ThirdPartyApis) (*v3.QueryRangeParamsV3, er
 		Legend:            "endpoints",
 		DataSource:        v3.DataSourceTraces,
 		StepInterval:      defaultStepInterval,
-		AggregateOperator: v3.AggregateOperatorCountDistinct,
+		AggregateOperator: v3.AggregateOperatorCount,
 		AggregateAttribute: v3.AttributeKey{
 			Key:      "http.url",
 			DataType: v3.AttributeKeyDataTypeString,
```

Comment:
why was this changed?

Reply:
I'm not sure if this runs on single thread or multiple threads, we don't need to change this if so why?

Reply:
Correctness is a non-negotiable requirement. The existing aggregation is countDistinct(http.url), which you're changing to count. These are not equivalent operations. However, countDistinct(span_id) would be the same as count because each row in the table already represents a unique span.

Reply:
> that the span ids are distinct, but there could be cases where the spans are not distinct, right?

When does this happen?

Reply:
> When does this happen?

I encountered this here https://github.com/SigNoz/signoz/issues/7286, more detailed issue is here https://github.com/SigNoz/signoz/issues/7281

Reply:
Yes, as you can see this is not specific to your current feature implementation but rather points to a broader issue that needs to be addressed separately.

Reply:
yes, can you re-review this PR?